packages/sdks/typescript/src/versions/LatitudeSdkV1.ts

```diff
@@ -0,0 +1,3 @@
+import { LatitudeSdk } from '$sdk/versions/LatitudeSdk'
+
+export class LatitudeSdkV1 extends LatitudeSdk { }
```

Comment:
The point of all these changes is to try to do a patch version in the SDK because current sdk users should not see any changes.